x-pack/plugins/actions/server/saved_objects/index.ts

```diff
@@ -22,7 +22,7 @@ export function setupSavedObjects(
     name: ACTION_SAVED_OBJECT_TYPE,
     hidden: true,
     namespaceType: 'single',
-    mappings: mappings.action,
+    mappings: mappings.action as SavedObjectsTypeMappingDefinition,
```

Comment:
mappings imported from `json` file are inferred with `{ type: string` since we cannot enforce immutability with `as const`, so I had to cast the type manually